src/visualstates/parsers/cppparser.py

```diff
@@ -87,15 +87,38 @@ def parseVariables(variableStr):
             initialValue = None
             if varLine.find('=') >= 0:
                 # if there is initial value
-                varName = varLine[varLine.find(' ')+1:varLine.find('=')].strip()
-                initialValue = varLine[varLine.find('=')+1:].strip()
+                varName = varLine[varLine.find(' ') + 1:varLine.find('=')].strip()
+                initialValue = varLine[varLine.find('=') + 1:].strip()
             else:
-                varName = varLine[varLine.find(' ')+1:].strip()
+                varName = varLine[varLine.find(' ') + 1:].strip()
 
             types.append(varType)
             varNames.append(varName)
             initialValues.append(initialValue)
 
+        for param in params:
```

Comment:
@okanasik Do we need changes in CPP files? As we have completely shifted to Python, maybe we can clear all the C++ related files and options on the GUI.

Reply:
Sure, we need changes for CPP and we still support CPP. I do not remember the time we decided to drop the support for C++.